lib/memory/index.js

```diff
@@ -58,13 +58,13 @@ module.exports = function MemoryStore(data_set) {
 
   const update = R.curry((bucket, id, data) =>
 
-      getById(bucket, id)
+    getById(bucket, id)
 
     .tap(R.when(R.isNil, () => throwItemNotFound(bucket, id)))
 
     .then(() => { store[bucket][id] = data })
 
-    .return(id)
+    .return(1)
```

Comment:
Hmm... the mysql update should be changed to return the identifier.  There's no reason to return the affected row count since it's never possible to update > 1.  The only concern would be a zero diff change, and in that case the programmer should check for that before they apply the update.  Any other issue will result in an exception being thrown from the MySQL driver.